### PR TITLE
Disable specific vendors

### PR DESCRIPTION
**Pull request by dha0stw**
add a snippet to read configure file and affect renderer to disable specific vendors.
e.g. avoid jquery conflict, but inject jquery library manually

### CODE REVIEW

File: src/LaravelDebugbar.php [hunk 1]
```diff
@@ -353,6 +353,11 @@ function ($query, $bindings, $time, $connectionName) use ($db, $queryCollector)
         }
 
         $renderer = $this->getJavascriptRenderer();
+        $disableVendors = $this->app['config']->get('debugbar.disableVendors', []);
+        if (!empty($disableVendors)) {
+            foreach($disableVendors as $vendor)
```
Comment (GrahamCampbell) [inline src/LaravelDebugbar.php]: missing braces

**Comment by dha0stw** [comment]
thanks, @GrahamCampbell !

File: config/debugbar.php [hunk 2]
```diff
@@ -51,6 +51,19 @@
 
     /*
      |--------------------------------------------------------------------------
+     | Disable Vendors
+     |--------------------------------------------------------------------------
+     |
+     | Could be easily disable specific vendor by setting following array.
+     | you should inject jquery library menually.
```
Comment (GrahamCampbell) [inline config/debugbar.php]: missing capital letter

File: config/debugbar.php [hunk 3]
```diff
@@ -51,6 +51,19 @@
...
+     | Options: 'jquery', 'highlightjs', 'fontawesome'
```
Comment (GrahamCampbell) [inline config/debugbar.php]: missing fullstop

**Comment by barryvdh** [comment]
Reverted because of incorrect syntax

File: src/LaravelDebugbar.php [hunk 4]
```diff
@@ -353,6 +353,12 @@ function ($query, $bindings, $time, $connectionName) use ($db, $queryCollector)
         }
 
         $renderer = $this->getJavascriptRenderer();
+        $disableVendors = $this->app['config']->get('debugbar.disableVendors', []);
+        if (!empty($disableVendors)) {
+            foreach($disableVendors as $vendor) {
+                $renderer->disableVendor($vendor);
+            {
```
Comment (barryvdh) [inline src/LaravelDebugbar.php]: Wrong accolade

**Pull request merged**